### PR TITLE
Recommended updates to README.md

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

**Comment by brianamarie** [comment]
@QuincyLarson These look good to me! 🚢  @atjonathan I know you helped organize this repo, what do you think?

File: README.md [hunk 1]
```diff
@@ -26,7 +30,7 @@ Video challenges are a segment of Free Code Camp's curriculum for full stack web
   - Save your changes and open a pull request comparing to master.
 3. Record your video
   - Set up a whiteboard and record your video 
-4. Comment in your PR with the link to your video. `@` QuincyLarson and brianamarie
+4. In your pull request, add a link to your video and @-mention QuincyLarson and brianamarie.
```
Comment (atjonathan) [inline README.md]: Lol, could you add me?!!

**Comment by atjonathan** [comment]
Other than that :joy:
#### LGTM :+1:

**Comment by QuincyLarson** [comment]
@atjonathan added you in my place.

**Comment by atjonathan** [comment]
Thanks, you can keep you though! :joy:

**Pull request merged**